.github/workflows/dotnetcore.yml

```diff
@@ -22,14 +22,18 @@ jobs:
     runs-on: windows-latest
     continue-on-error: true
     name: "Build and run unit tests"
+
+    strategy: 
+      matrix:
+        dotnet-version:[ '6.0.x', '7.0.x', '8.0.x', '9.0.x' ]
```

Comment:
this seems to help against any issues w/the frameworks not running properly, there seemed to be an issue w/the wrong framework running on the OIDC tests

Reply:
I tried installing all frameworks yesterday, it didn't help. But if this works, then it works...

Reply:
it worked when I did a test PR

Reply:
The run failed?
https://github.com/AzureAD/microsoft-identity-web/actions/runs/11822912264

Reply:
The run failed?
https://github.com/AzureAD/microsoft-identity-web/actions/runs/11822912264

Reply:
That's not what it showed. and then I changed the base branch so I don't have proof.